packages/design-system/src/components/MonthPicker/MonthPicker.test.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import MonthPicker from './MonthPicker';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+const defaultProps = {
+  name: 'months',
+  label: 'Months',
+  selectAllText: 'Select all',
+  clearAllText: 'Clear all',
+};
+
+function renderMonthPicker(customProps = {}) {
+  const props = { ...defaultProps, ...customProps };
+  return render(<MonthPicker {...props} />);
+}
+
+describe('MonthPicker', () => {
+  it('renders a snapshot', () => {
+    const { asFragment } = renderMonthPicker();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders inverse style', () => {
+    renderMonthPicker({ inversed: true });
+
+    const buttons = screen.getAllByRole('button');
+    const checkboxes = screen.getAllByRole('checkbox');
+
+    buttons.forEach((button) => expect(button.classList).toContain('ds-c-button--on-dark'));
```

Comment:
The way I've seen this done in most cases has been to use `toHaveClass`, but this seems fine, just something I noticed.

Reply:
Oh, I was just following what I thought was convention in our DS. Pushing an update to fix this!